### DISCUSSION TITLE
can not submit an empty date field to my query

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#1568] [Help] [answered]
I found a bug. On the new BB updated yesterday, I can no longer submit an empty date field to my query. an empty field should fallback on the the parameters default value, however it is just saying error.
![image](https://user-images.githubusercontent.com/80058238/119701420-29311e00-be09-11eb-9d27-9ef815671a0b.png)

**Comment by aptkingston** [comment, marked as answer]
Hi @AndrewJRamay. Thanks for reporting. This has already been raised as an issue (https://github.com/Budibase/budibase/issues/1504), so you can keep track of progress there.

**Reply by KernOil** [reply]
Thank you!
is there a better place for me to report bugs or just like this?

**Reply by aptkingston** [reply]
If you think you find a bug then it would be better to raise it as an issue. For anything else, discussions is the way to go.
